index.js

```diff
@@ -2,9 +2,11 @@
 
 var exec = require('child_process').exec;
 var plist = require('simple-plist');
+var CertDownloader = require ('cert-downloader');
```

Comment:
Can't you just instantiate it immediately here? Or make the module return an instance if not.

```js
var certificate = new require('cert-downloader')();
```

NB: haven't tested this, would just look a lot cleaner imo.